arrow-libs/optics/arrow-optics/src/main/kotlin/arrow/optics/std/tuple.kt

```diff
@@ -1,239 +0,0 @@
-package arrow.optics
-
-import arrow.core.Tuple10
-import arrow.core.Tuple2
-import arrow.core.Tuple3
-import arrow.core.Tuple4
-import arrow.core.Tuple5
-import arrow.core.Tuple6
-import arrow.core.Tuple7
-import arrow.core.Tuple8
-import arrow.core.Tuple9
-import arrow.core.toT
-
-/**
- * [PLens] to focus into the first value of a [arrow.core.Tuple2]
- */
```

Comment:
Nothing in this file can be removed since present in 0.11.0. All these methods need to be deprecated in favor of their `@JvmStatic` version on `Lens` `Companion`.